Svc/PrmDb/test/ut/PrmDbImplTester.cpp

```diff
@@ -346,234 +351,160 @@ namespace Svc {
         PrmDbGTestBase::init();
     }
 
+    PrmDbImplTester* PrmDbImplTester::PrmDbTestFile::s_tester = nullptr;
 
-void PrmDbImplTester::runFileReadError() {
```

Comment:
What if anything is replacing what this deleted code does?

Reply:
`runFileReadError` still exists.  Instead of doing copy-and-past of 4 large blocks of code, I reduced the 4 read steps into a `for` loop as all but the record type was the same.  Makes it easier to maintain when you need to update the order.

Reply:
Not exactly accurate, it is reduced to several for loops whose purpose is to setup a failed read after `i` successful reads. The process is the same for each so a for loop is appropriate.

An example is here: https://github.com/nasa/fprime/blob/b2fd0791b536489895fa6f434776be50a092c9e4/Svc/PrmDb/test/ut/PrmDbImplTester.cpp#L413-L438

Reply:
Thanks for the explanation. I agree with the changes.